### ISSUE TITLE
Demo > Lack of Alt Text Button Image Toolbar

### ISSUE THREAD

**Issue by duracell80** [liferay/alloy-editor#1370]
See title

**Comment by duracell80** [comment]
Again I'm pointing out a regression in the demo that speaks to a wider perception that this editor just is not as good as the one it is built on.

**Comment by julien** [comment]
Hi @duracell80, 

If you think that alloy-editor is not the correct editor for your use case, you can always configure DXP/liferay-portal to use CKEditor by adding these settings in your `portal-ext.properties`

```properties
jeditor.wysiwyg.default=ckeditor
editor.wysiwyg.portal-web.docroot.html.portlet.blogs.edit_entry.jsp=ckeditor
editor.wysiwyg.portal-web.docroot.html.portlet.journal.edit_article_content.jsp=ckeditor
editor.wysiwyg.portal-impl.portlet.ddm.text_html.ftl=ckeditor
```

**Comment by duracell80** [comment]
Please stop giving that as a solution.

**Comment by julien** [comment]
I'm just trying to help, based on your previous comment

> Again I'm pointing out a regression in the demo that speaks to a wider perception that this editor just is not as good as the one it is built on.

**Comment by duracell80** [comment]
That's all we get from Liferay though ... "You don't have to use it" ... Errrrrrr yeah we do. Particularly in fragments.
